.github/.gitmessage

```diff
@@ -0,0 +1,20 @@
+# Title: Summary, imperative, start upper case, don't end with a period
+# No more than 50
+Replace this title line with an appropriate title
+# _______________ 50  characters _______________ #
+# Pick one (below) and use it (remove "#") and change: feat/fix, scope, description
+# feat: description
+# feat[scope]: description
+# fix: description
+# fix[scope]: description
```

Comment:
Do we want to pop in the other prefixes for conventional commits? Notably ones we use in practice:

- `docs`
- `chore`
- `refactor`

Full list [located here](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)

Reply:
Do we have a short list? Or just include them all? I tend to get a little lost in the classifications and try and restrict myself to either "feature" or "non-feature".

Reply:
If we are going to add these I'd also like to add a bit of detail when to use which. Specifically, "chore" and "refactor"; how are those different from "feat" or "fix". I am assuming these:

- `docs` - changes made are only to improving documentation and specifically not changing any client-facing functionality
- `build` - changes made are only to the system that builds artifacts and specifically not changing any client-facing functionality

Reply:
If you look at the tags for this repo, they all have the spec descriptions added to them. Also in that link Brandon posted.

Reply:
I added a link in the file to the labels page of the repo so as to not duplicate documentation and allow it to drift eventually.